server.go

```diff
@@ -202,20 +188,63 @@ func getRequestBody(input string) (*http.Request, error) {
 	inputs = append(inputs, data)
 	reqBody := RequestBody{inputs}
 	fmt.Print(reqBody)
-	//reqBodyBytes, err := json.Marshal(reqBody)
-	//if err != nil {
-		//return nil, err
-	//}
 
 	b := new(bytes.Buffer)
-  err := json.NewEncoder(b).Encode(reqBody)
+	err := json.NewEncoder(b).Encode(reqBody)
 	req, err := http.NewRequest("POST", clarifaiApi, b)
 	fmt.Print(req)
 	if err != nil {
 		return nil, err
 	}
 
-	req.Header.Add("Authorization", `Bearer ThzzqYyVARtJdLbTQDwfRpa1FOk8w6`)
+	req.Header.Add("Authorization", `Bearer ` + clarifai_token.Token)
 	req.Header.Add("Content-Type", `application/json`)
 	return req, nil
 }
+
+func getAccesToken() AccessToken {
```

Comment:
The function will be too big imo, I like having the smaller function possible doing one thing, then it's easier to test and modify code, so here one makes request, the other extracts the data we need